.github/renovate.json5

```diff
@@ -237,9 +231,7 @@
         "go.sum"
       ],
       matchBaseBranches: [
-        "v1.18",
-        "v1.17",
-        "v1.16"
+        "!main",
```

Comment:
Amazing. I never thought about this because I thought it would be applied to all branches but it's only the subset of the branches defined under baseBranches on line 76